smslibrary/src/main/java/com/eis/communication/Peer.java

```diff
@@ -1,17 +1,18 @@
 package com.eis.communication;
 
-import androidx.annotation.NonNull;
-
 import java.io.Serializable;
 
 /**
  * Represents a user in the network
+ *
  * @param <A> The type of address used by the network
+ * @see Serializable
  */
 public interface Peer<A> extends Serializable {
 
     /**
      * Retrieves the peer's address
+     *
```

Comment:
I mean, look at this, another edited file just to add some documentation, could have been put in another pull request

Reply:
I know but “già che c’ero”